### PR TITLE
R4R: user guide for bank

### PR DESCRIPTION
**Pull request by chengwenxi**

### CODE REVIEW

File: docs/features/bank.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        "memo": ""
+      }
+    }
+    ```
+    Save the result to a file.
+    
+5. Broadcast transactions
+
+    Broadcast signed transactions generated offline, using the final generated file in step 4:
+    ```bash
```
Comment (HaoyangLiu) [inline docs/features/bank.md]: > Broadcast signed transactions generated offline, using the final generated file in step 4:
How about change the sentence like this:
```
Broadcast offline signed transaction. Here you just use the transaction generated by above sign command. Of course, you can generate your signed transaction by any methods. Just make sure your signature and public key are amino encoding.
```

File: docs/features/bank.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    iriscli bank broadcast [file]
+    ```
+    The transaction will broadcast and executed in IRISnet.
+
```
Comment (HaoyangLiu) [inline docs/features/bank.md]: > The transaction will broadcast and executed in IRISnet.
Remove this line, or change it as:
```
The transaction will be broadcasted and executed in blockchain.
```

File: docs/features/bank.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+          "gas": "200000"
+        },
+        "signatures": null,
+        "memo": ""
+      }
+    }
+    ```
+    Save the result to a file.
+    
+    Send signature transaction:
```
Comment (HaoyangLiu) [inline docs/features/bank.md]: > Send signature transaction:

Add signature to this transaction

File: docs/features/bank.md (hunk 1 above)
Comment (chengwenxi) [reply docs/features/bank.md]: Great！

**Pull request merged**